koku/sources/openapi.json

```diff
@@ -0,0 +1,108 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "The Sources Status API.  You can find out more about     Cost Management at [https://github.com/project-koku/](https://github.com/project-koku/).",
+    "version": "1.0.0",
+    "title": "Sources Status API",
+    "license": {
+      "name": "AGPL-3.0",
+      "url": "https://opensource.org/licenses/AGPL-3.0"
+    }
+  },
+  "tags": [
+    {
+      "name": "Source Status",
+      "description": "Operations about report downloads"
+    }
+  ],
+  "paths": {
+    "/source_status/": {
+      "get": {
+        "summary": "Get Source Status",
+        "description": "Return whether or not a source is correctly set up.",
+        "parameters": [
+          {
+            "name": "source_id",
+            "in": "query",
+            "description": "The source_id",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "example": 1
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The status of the source",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/StatusGetResponse"
+                },
+                "examples": {
+                  "Available": {
+                    "value": {
+                      "availability_status": "available",
+                      "availability_status_error": ""
+                    }
+                  },
+                  "NotAvailable": {
+                    "value": {
+                      "availability_status": "unavailabile",
+                      "availability_status_error": "Bucket cost-usage-buckett could not be found with arn:aws:iam::589173575009:role/CostManagement."
```

Comment:
```suggestion
                      "availability_status_error": "Bucket cost-usage-bucket could not be found with arn:aws:iam::999999999999:role/CostManagement."
```

Reply:
in this example, the error was a real error that I intentionally spelled bucket with two 't's in order to cause the error to occur, otherwise the error would not occur. 
I can rename it to something like "Bucket nonexistantbucketexamplename could not(...)" etc

Reply:
resolved as per changing the name to nonexistantbucketexample